lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -16,16 +16,33 @@
   font-weight: normal;
 }
 
+@mixin italic() {
```

Comment:
I think the value of these is debatable.

Reply:
> I think the value of these is debatable.

I know but we use them and we use them consistently so we may as well have full coverage